src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -3500,6 +3500,9 @@ def create_functionapp(cmd, resource_group_name, name, storage_account, plan=Non
     client = web_client_factory(cmd.cli_ctx)
 
     if vnet or subnet:
+        if environment is not None:
+            raise UnsupportOperationError('vnet paramter and managed environment parameter canot be passed in at the same time', 
+            'Please config your vnet when create the container app managed environment.')
```

Comment:
1. It looks like the `az containerapp env update` command does not consume the vnet information. Looks like we might need to do that change as well.
2. I think we might need to rephrase this to say "Please configure your virtual network by updating your container app managed environment with the `az containerapp env update` command." since the current message gives the impression that they can only config vnet information when creating the managed environment (but at this point they would have already created the managed environment). 
3. We could use `ArgumentUsageError` which already exists instead of `UnsupportOperationError`. I'm good with either one. Just wanted to point out that there is already an AzCLI error type that we could use.

Reply:
1. I think we can leave that to container app team. 
2. I just updated the message using the one send by the PM
3. I update the `create_functionapp` to use `ArgumentUsageError`, for add and remove vnet commands I think `UnsupportOperationError` is more suitable here. Let me know what you think.